rust-src/aggregate_sig/src/aggregate_sig.rs

```diff
@@ -228,7 +277,7 @@ mod test {
     use super::*;
     use pairing::bls12_381::Bls12;
     use rand::{rngs::StdRng, thread_rng, SeedableRng};
-    use std::convert::TryFrom;
+    use std::{convert::TryFrom, slice::from_ref};
```

Comment:
Please use this qualified and don't import `from_ref` like this. It is going to be much clearer when it is used.